dotcom-rendering/src/web/layouts/FrontLayout.tsx

```diff
@@ -147,6 +147,14 @@ export const FrontLayout = ({ front, NAV }: Props) => {
 					// There are some containers that have zero trails. We don't want to render these
 					if (trails.length === 0) return null;
 
+					// This is a legacy container used to add palette styling on Frontend. DCR ignores it
+					if (
+						collection.displayName ===
+						'Palette styles new do not delete'
```

Comment:
is there a case to be made  for filtering this  container  out  before `map`-ing  on the list? It might  enable  us to simplify some of the index logic below, e.g. https://github.com/guardian/dotcom-rendering/blob/d65f5aac0fc48e383065f931ee517c21f40d5e10/dotcom-rendering/src/web/layouts/FrontLayout.tsx#L183-L192

Reply:
Nice idea! I think this makes a lot of sense. I think this would work well as an enhancer.

I've got another thrasher related PR pending so I will either put this change in there or do it after that one